lib/iris/options.py

```diff
@@ -0,0 +1,209 @@
+"""
+Control runtime options of Iris.
+
+"""
+from __future__ import (absolute_import, division, print_function)
+from six.moves import (filter, input, map, range, zip)  # noqa
+
+from multiprocessing import cpu_count
+from multiprocessing.pool import ThreadPool
+import re
+import warnings
+
+import dask
+import distributed
+
+
+class Parallel(object):
+    """
+    Control dask parallel processing options for Iris.
+
+    """
+    def __init__(self, scheduler=None, num_workers=None):
+        """
+        Set up options for dask parallel processing.
+
+        Currently accepted kwargs:
+
+        * scheduler:
+            The scheduler used to run a dask graph. Must be set to one of:
+
+            * 'threaded': (default)
+                The scheduler processes the graph in parallel using a
+                thread pool. Good for processing dask arrays and dataframes.
+            * 'multiprocessing':
+                The scheduler processes the graph in parallel using a
+                process pool. Good for processing dask bags.
+            * 'async':
+                The scheduler runs synchronously (not in parallel). Good for
+                debugging.
+            * The IP address and port of a distributed scheduler:
+                Specifies the location of a distributed scheduler that has
+                already been set up. The distributed scheduler will process the
+                graph.
+
+            For more information see
+            http://dask.pydata.org/en/latest/scheduler-overview.html.
+
+        * num_workers:
+            The number of worker threads or processess to use to run the dask
+            graph in parallel. Defaults to 1 (that is, processed serially).
+
+            .. note::
+                The value for `num_workers` cannot be set to greater than the
+                number of CPUs available on the host system. If such a value is
+                requested, `num_workers` is automatically set to 1 less than
+                the number of CPUs available on the host system.
+
+            .. note::
+                Only the 'threaded' and 'multiprocessing' schedulers support
+                the `num_workers` kwarg. If it is specified with the `async` or
+                `distributed` scheduler, the kwarg is ignored:
+
+                * The 'async' scheduler runs serially so will only use a single
+                worker.
+                * The number of workers for the 'distributed' scheduler must be
+                defined when setting up the distributed scheduler. For more
+                information on setting up distributed schedulers, see
+                https://distributed.readthedocs.io/en/latest/index.html.
+
+        Example usages:
+
+        * Specify that we want to load a cube with dask parallel processing
+        using multiprocessing with six worker processes::
+
+        >>> iris.options.parallel(scheduler='multiprocessing', num_workers=6)
+        >>> iris.load('my_dataset.nc')
```

Comment:
To be consistent with the other code examples, you should remove the python interpreter prompts i.e. the `>>>` and the `...`